app/ts-cli/cmd/export.go

```diff
@@ -0,0 +1,126 @@
+package cmd
+
+import (
+	"bufio"
+	"encoding/json"
+	"flag"
+	"github.com/openGemini/openGemini/app/ts-cli/geminicli"
```

Comment:
Use `goimports-reviser` to sort packages, such as: `goimports-reviser -project-name none-pjn ./...`